### PR TITLE
Add "Fortune and Folly" encounters

### LINKED ISSUE #1274: Fortune and Folly encounters
As discussed over on the mythos busters discord, we (coldtoes over on discord and me) want to contribute _Fortune and Folly_ encounters to the database. I'm opening this as a tracking issue to have a more central place to discuss details specific to this set.

### PR DESCRIPTION
**Pull request by fspoettel**
closes #1274

This adds all _Fortune and Folly_ encounter cards. I tested and proofread changes in a local arkhamdb instance before opening this PR. I also posted a zip file with id mapped images to discord.

Note for reviewers: as discussed in the issue, I added card icons to the card text and treated these encounters as individual cards. While typing, I discovered that the cards have individual `encounter_positions` and different _Patrol_ instructions, so this seems like the correct choice to me.

Note for reviewers: I did not update code `88034a` to `88034` although this would be consistent with the rest of the cards. Not sure about backwards-compat.

### CODE REVIEW

**Comment by drawntotheflame** [comment]
I'm on the case - will get you feedback as soon as I'm able.

File: types.json [hunk 1]
```diff
@@ -42,5 +42,9 @@
     {
         "code": "treachery",
         "name": "Treachery"
+    },
+    {
```
Comment (drawntotheflame) [inline types.json:46]: This has already been added by the TSK merge, so you can remove this. (I don't think it causes a problem if it's re-added either...)

**Review by drawntotheflame — COMMENTED** [review]
Looking over this, I think multiple entries for cards that are identical barring their suit is maybe not the way to go? Even with a shared code, it means there'll be multiple entries for the same card. Could we move it so that we record suits with a | dividing them where one card has multiple versions with different suits?

Edit: sorry, I see what you mean about different patrol instructions. Hmm, it's a tricksy one. Let me have a think.

**Review by drawntotheflame — COMMENTED** [review]
Good job! lots to do!

**Comment by fspoettel** [comment]
@drawntotheflame any updates?

**Comment by fspoettel** [comment]
Hey @drawntotheflame, is there anything you like to change here or can we merge this? It's been a few months 😅

**Comment by drawntotheflame** [comment]
Hi @fspoettel thanks for the tag. Silence on my end because of the tricksiness of different patrol instructions on otherwise identical cards. @zzorba any views on this PR, when you have a moment?

**Comment by zzorba** [comment]
I think we should probably make a/b/c/ versions of the cards, similar to what we did in some of the Scarlet Keys (where the same card front has 3 different backs, despite having the same printed #).

**Comment by fspoettel** [comment]
I'm happy to make the change if everyone is happy with that solution (@drawntotheflame ?).

**Comment by drawntotheflame** [comment]
yes, it sounds very positive!

**Comment by fspoettel** [comment]
updated the ids and verified the change in a local arkhamdb instance. this should be ready now @drawntotheflame .

**Comment by drawntotheflame** [comment]
@Kamalisk I think this can go live.

**Comment by fspoettel** [comment]
@drawntotheflame or @Kamalisk I have to encounter images mapped out to the ids for these, feel free to DM me on discord if you want me to zip & send them over.

**Pull request merged**